src/sidecar_comms/form_cells/base.py

```diff
@@ -22,7 +22,7 @@
 Finally, the backend implementation of form cells use a discriminator pattern. You can create a
 form cell from a dictionary that is coming in from the frontend. Example:
 
-model = pydantic.parse_obj_as(FormCell, {"input_type": "datetime", "value": "2021-01-01"})
+model = pydantic.TypeAdapter(FormCell).validate_python({"input_type": "datetime", "value": "2021-01-01"})
```

Comment:
Probably should just construct the TypeAdapter just once at module level, then reuse it. No need to generate a new TypeAdapter instance each call.